internal/core/metadata/operators/device_service/update_test.go

```diff
@@ -0,0 +1,307 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package device_service
+
+import (
+	goErrors "errors"
```

Comment:
I like this approach to aliasing the Go error import. I have been using `errors2` to alias our error packages, but I like this more as I think we should really be using our own errors types more often, which means aliasing the Go errors would make more sense.